rules/prettier.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
   arrowParens: 'always',
-  parser: 'babel',
   printWidth: 80,
   semi: false,
   singleQuote: true,
   tabWidth: 2,
   trailingComma: 'es5',
+  quoteProps: 'preserve',
```

Comment:
We has this defined as a rule in eslint, so to avoid conflicts we tell prettier to leave these.